hawkular-inventory-api/pom.xml

```diff
@@ -78,7 +79,6 @@
 
   <build>
     <finalName>hawkular-${project.artifactId}-${project.version}</finalName>
```

Comment:
There's no need for this anymore as this should be the same as the default name now.